controllers/ibmpowervsimage_controller.go

```diff
@@ -121,7 +121,7 @@ func (r *IBMPowerVSImageReconciler) reconcile(ctx context.Context, cluster *v1be
 		job, err := imageScope.IBMPowerVSClient.GetJob(jobID)
 		if err != nil {
 			imageScope.Info("Unable to get job details")
-			return ctrl.Result{RequeueAfter: 2 * time.Minute}, err
```

Comment:
any specific reason for reducing the reque time?

Reply:
No specific reason, just that sometimes even when the image is imported successfully it takes longer to reflect in status because of reque time (So reducing the reque time for quicker status reflection)